collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -246,6 +253,14 @@ protected void addQuestionMediaLayout(View v) {
         addView(v, params);
     }
 
+    public Bundle getState() {
+        saveState();
+        return state;
+    }
+
+    protected void saveState() {
+        state = new Bundle();
```

Comment:
Can you add an `@OverridingMethodsMustInvokeSuper` annotation here? As it stands, if a child doesn't call `super.saveState` or handle managing the `state` variable themselves, then state is never set. Its unclear which is the expected behavior for overriding.